lms/templates/problem.html

```diff
@@ -15,13 +15,13 @@ <h2 class="problem-header">
     <input type="hidden" name="problem_id" value="${ problem['name'] }" />
 
     % if check_button:
-    <button class="check ${ check_button }" data-checking="${ check_button_checking }" data-value="${ check_button }"><span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span></button>
+    <button class="check ${ check_button }" data-checking="${ check_button_checking }" data-value="${ check_button }"><span class="check-label button-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span></button>
     % endif
     % if reset_button:
-    <button class="reset" data-value="${_('Reset')}">${_('Reset')}<span class="sr"> ${_("your answer")}</span></button>
+    <button class="reset" data-value="${_('Reset')}"><span class="button-label">${_('Reset')}</span><span class="sr"> ${_("your answer")}</span></button>
     % endif
     % if save_button:
-    <button class="save" data-value="${_('Save')}">${_('Save')}<span class="sr"> ${_("your answer")}</span></button>
+    <button class="save" data-value="${_('Save')}"><span class="button-label">${_('Save')}</span><span class="sr"> ${_("your answer")}</span></button>
```

Comment:
Since these buttons should look the same, I recommend just updating the main `button` style. But if that isn't possible, I'm okay with this approach so long as __all__ buttons have the same markup.